io/solarnet.py

```diff
@@ -0,0 +1,170 @@
+"""
+Python tools to read and write 'SOLARNET' cubes
+
+Written by Carlos Diaz (ISP/SU 2019)
+
+"""
+import astropy.io.fits as fits
+import numpy as np
+import os
+import sys
+
+# ========================================================================
+def read(filename):
+    """Reads the data of a 'SOLARNET' cube.
+
+    Arguments:
+        filename: name of the data cube
+
+    Returns:
+        1D wavelength array in nm.
+
+    Authors: Carlos Diaz (ISP/SU 2019)
+    """
+    return fits.getdata(filename, ext=0)
+
+
+# ========================================================================
+def write(filename, d):
+    """Reads the data as a normal fits (without proper SOLARNET header)
+
+    Arguments:
+        filename: name of the data cube
+        d: data on memory
+
+    Authors: Carlos Diaz (ISP/SU 2019)
+    """
+    io = fits.PrimaryHDU(d)
+    io.writeto(filename, overwrite=True)
+    return
+
+
+# ========================================================================
+def get_wav(filename):
+    """Reads the wavelength information of a 'SOLARNET' cube.
+
+    Arguments:
+        filename: name of the data cube
+        - It is assumed that spectral sampling does not change over time
+
+    Returns:
+        1D wavelength array in nm.
+
+    Example:
+        get_wav(filename, fulltime=False, timeformat=False)
+
+    Authors: Carlos Diaz (ISP/SU 2019)
+    """
+    io = fits.open(filename)
+    # Wavelength information for all wavelength points in the first time frame
+    wav_output = io[1].data[0][0][0,:,0,0,2]
+    return wav_output
+
+
+# ========================================================================
+def seconds2string(number):
+    """Converts time in seconds to 'HH:MM:SS' format
+
+    Arguments:
+        number: value in seconds
+
+    Returns:
+        Time in 'HH:MM:SS' format
+
+    Authors: Carlos Diaz (ISP/SU 2019)
+    """
+    hour = number//3600.
+    minute = (number%3600.)//60.
+    seconds = (number-(hour*3600.+minute*60))/1.
+    string_output = '{0}:{1}:{2:09.6f}'.format(int(hour),int(minute),seconds)
+    return string_output
+
+
+# ========================================================================
+def get_time(filename, fulltime=False, timeformat=False):
```

Comment:
Maybe change `timeformat` keyword to `utc` for clarity.

Reply:
Thanks. Solved in last commit.